src/Mutations/User.tsx

```diff
@@ -31,15 +31,15 @@ export const GET_PROFILE = gql`
             startDate
           }
         }
-        cohort {
-          name
-          program {
-            name
-          }
-          phase {
-            name
-          }
-        }
+        # cohort {
+        #   name
+        #   program {
+        #     name
+        #   }
+        #   phase {
+        #     name
+        #   }
+        # }
         program {
```

Comment:
if these commented lines of code are not needed can you remove them/

Reply:
@elijahladdie, Done